jboss/container/config/cd/19.0/standalone-openshift.xml

```diff
@@ -460,13 +459,6 @@
         <subsystem xmlns="urn:wildfly:microprofile-health-smallrye:2.0" security-enabled="false" empty-liveness-checks-status="${env.MP_HEALTH_EMPTY_LIVENESS_CHECKS_STATUS:UP}" empty-readiness-checks-status="${env.MP_HEALTH_EMPTY_READINESS_CHECKS_STATUS:UP}"/>
         <subsystem xmlns="urn:wildfly:microprofile-jwt-smallrye:1.0"/>
         <subsystem xmlns="urn:wildfly:microprofile-metrics-smallrye:2.0" security-enabled="false" exposed-subsystems="*" prefix="${wildfly.metrics.prefix:jboss}"/>
-        <subsystem xmlns="urn:jboss:domain:modcluster:5.0">
```

Comment:
We didn't remove the modcluster in cd19. We should keep it in the reference file.

Reply:
@rhusar @luck3y why this jfdenise objection was not addressed?

Reply:
@mchoma, it was, I think the git UI doesn't show it though: https://github.com/jboss-container-images/jboss-eap-modules/blob/master/jboss/container/config/cd/19.0/standalone-openshift.xml#L463

I had edited the PR to add those lines back to the config, then pushed it.

Reply:
LGTM, thanks again @luck3y